src/components/NcPopover/NcPopover.vue

```diff
@@ -257,6 +425,35 @@ export default {
 	},
 
 	methods: {
+		logicalPlacement(placement) {
+			if (placement === 'start') {
+				return 'left'
+			} else if (placement === 'end') {
+				const trigger = this.getPopoverTriggerContainerElement() ?? this.$refs.popover?.$el
+				return this.logicalDirection(trigger, 'right', 'left')
+			}
+			return placement
+		},
+
+		/**
+		 * @param {HTMLElement} element The HTML element to check
+		 * @param {string} ltr LTR alternative
+		 * @param {string} rtl RTL alternative
+		 * @returns {string} The selected alternative
+		 */
+		logicalDirection(element, ltr, rtl) {
+			console.error(element, ltr, rtl)
+			if (!element) {
+				return ltr
+			}
+			const direction = window.getComputedStyle(element).direction
+			const { dir } = element
```

Comment:
Why not use `isRTL`? Do we need to override popover's dir?

Reply:
This requires changing placement after mounting the component

Reply:
Component can be placed in mixed content

Reply:
So basically we have to decide:
Do we want layout based on language only or layout based on surrounding layout.
E.g. if I set language to Arabic its RTL, but if my component is used in a LTR context (e.g. in a contenteditable context with English text around) then it would be RTL even if everything around is LTR (imagine a user bubble with context menu).

I am fine with using `isRTL` but then we have the limitation mentioned, just want to confirm this.

Reply:
removed for isRtl for now.